core/templates/dev/head/pages/topic_editor/questions/QuestionsTabDirective.js

```diff
@@ -24,13 +24,13 @@ oppia.directive('questionsTab', [
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/topic_editor/questions/questions_tab_directive.html'),
       controller: [
-        '$scope', '$http', '$q', '$uibModal', 'AlertsService',
+        '$scope', '$http', '$q', '$uibModal', '$window', 'AlertsService',
```

Comment:
Why add $window? Doesn't seem to be used...

Reply:
@nithusha21 this is causing an error preventing the creator dashboard to fully load - 
_Cause: undefined
$window is not defined
ReferenceError: $window is not defined_

Reply:
Done

Reply:
Actually I added it as I wanted to refresh the page after accept/reject. In that commit I hadn't made calls to the refresh function. The refresh is called now.